cli/src/generate/templates/Package.swift

```diff
@@ -0,0 +1,42 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "TreeSitterCAMEL_PARSER_NAME",
+    platforms: [.macOS(.v10_13), .iOS(.v11)],
+    products: [
+        .library(name: "TreeSitterCAMEL_PARSER_NAME", targets: ["TreeSitterCAMEL_PARSER_NAME"]),
+    ],
+    dependencies: [],
+    targets: [
+        .target(name: "TreeSitterCAMEL_PARSER_NAME",
+                path: ".",
+                exclude: [
+                    "binding.gyp",
+                    "bindings",
+                    "Cargo.toml",
+                    "corpus",
+                    "examples",
+                    "test",
+                    "grammar.js",
+                    "LICENSE",
+                    "Makefile",
+                    "package.json",
+                    "README.md",
+                    "script",
+                    "src/grammar.json",
+                    "src/node-types.json",
+                    "*.wasm",
```

Comment:
An exclude list is unnecessary for recent Swift versions. It is safe to just leave this array empty.